squarelet/organizations/tests/test_api.py

```diff
@@ -314,14 +314,28 @@ def test_list(self, api_client, mocker):
         size = 10
         entitlements = EntitlementFactory.create_batch(size)
         mocker.patch("stripe.Plan.create")
-        # make entitlements public by adding it to a public plan
         plan = PlanFactory(public=True)
         plan.entitlements.set(entitlements)
         response = api_client.get(f"/pp-api/entitlements/")
         assert response.status_code == status.HTTP_200_OK
         response_json = json.loads(response.content)
         assert len(response_json["results"]) == size
 
+    def test_subscribed(self, api_client, user):
+        """List entitlements for a subscribed user"""
+        size = 10
+        api_client.force_authenticate(user=user)
+        org = OrganizationFactory(admins=[user])
+        plan = PlanFactory(for_individuals=False, for_groups=True, public=False)
+        plan.private_organizations.add(org)
+        entitlements = EntitlementFactory.create_batch(size)
```

Comment:
You should create some entitlements which the user is not subscribed to, so the test is testing that those do *not* show up - otherwise subscribed could return extra entitlements and the test would miss it